From 2b626d42bcf67b21094f304d6eb3257afbc2c1b1 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Mon, 18 Aug 2014 19:56:46 -0400
Subject: [scsi] qla2xxx: Remove unnecessary printk_ratelimited from qla_nx2.c

Message-id: <1408391817-6389-59-git-send-email-cdupuis@redhat.com>
Patchwork-id: 87949
O-Subject: [PATCH RHEL 7.1 e-stor 58/69] qla2xxx: Remove unnecessary printk_ratelimited from qla_nx2.c
Bugzilla: 1089346
RH-Acked-by: Rob Evers <revers@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Chad Dupuis <chad.dupuis@qlogic.com>

scsi commit id b6ae5ba6c73f585ae03e4cc7c4c62bf131ff2f8d

Signed-off-by: Chad Dupuis <chad.dupuis@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/qla2xxx/qla_nx2.c b/drivers/scsi/qla2xxx/qla_nx2.c
index 26de177..26e260e 100644
--- a/drivers/scsi/qla2xxx/qla_nx2.c
+++ b/drivers/scsi/qla2xxx/qla_nx2.c
@@ -2397,8 +2397,6 @@ qla8044_minidump_process_rdmem(struct scsi_qla_host *vha,
   }
 
   if (j >= MAX_CTL_CHECK) {
-   printk_ratelimited(KERN_ERR
-       "%s: failed to read through agent\n", __func__);
    write_unlock_irqrestore(&ha->hw_lock, flags);
    return QLA_SUCCESS;
   }
-- 
1.7.1